package.json

```diff
@@ -114,7 +114,6 @@
     "jsonwebtoken": "^8.5.1",
     "jwt-decode": "^3.1.2",
     "moment": "^2.29.3",
-    "multer": "^1.4.4",
```

Comment:
Should have been removed when we remove the multipart form upload capability. Just noticed it because it has a CVE.